compiler/rustc_borrowck/src/diagnostics/explain_borrow.rs

```diff
@@ -245,8 +310,8 @@ impl<'tcx> BorrowExplanation<'tcx> {
                 from_closure: _,
             } => {
                 region_name.highlight_region_name(err);
-
                 if let Some(desc) = opt_place_desc {
+                    //FIXME: caused ui test failed
```

Comment:
This comment wouldn't be meaningful without the context of this pull request, so we should probably remove it - if you keep the code that you tried and we can see the failures then we might be able to resolve it.